app/models/provided_access_token.rb

```diff
@@ -6,5 +6,6 @@ class ProvidedAccessToken < ApplicationRecord
 
   default_scope -> { order(expires_at: :desc)}
   scope :valid, -> { where.has { expires_at > Time.now.utc } }
-
+  scope :expired, ->(time = Time.now.utc) { where.has { expires_at <= time } }
+  scope :long_expired, -> { expired(1.day.ago.end_of_day) }
```

Comment:
I think this is not needed

Reply:
The idea was leaving the most recent records for some auditing or debugging. Do you think we should use `expired` directly instead?

Reply:
I think that's what Hery is suggesting is like he told me for this:
https://github.com/3scale/porta/blob/411771b501df2b723c6728e9057ce39b0c73e791/app/models/account/states.rb#L2
https://github.com/3scale/porta/blob/411771b501df2b723c6728e9057ce39b0c73e791/app/models/account/states.rb#L97-L99

Reply:
So by the `expired` directly, and if nothing is sent, we have a default (in this case `1.day.ago.end_of_day`)